status.md

```diff
@@ -1,11 +1 @@
-This document is a [Working Group Note](http://www.w3.org/Consortium/Process/tr#WGNote). 
 
-It builds on WCAG-EM, which was developed by the [WCAG 2.0 Evaluation Methodology Task Force (Eval TF)](http://www.w3.org/WAI/ER/2011/eval/eval-tf), a joint task force of the [Web Content Accessibility Guidelines Working Group (WCAG WG)](http://www.w3.org/WAI/GL/) and [Evaluation and Repair Tools Working Group (ERT WG)](http://www.w3.org/WAI/ER/). It provides informative guidance on evaluation in accordance with [Web Content Accessibility Guidelines (WCAG) 2.0](http://www.w3.org/WAI/intro/wcag).
```

Comment:
This is still relevant.

Reply:
added it back slight rewording of initial paragraph